clamav-bytecode-compiler/1.4/ubuntu/Jenkinsfile

```diff
@@ -71,7 +71,7 @@ node('docker') {
 
                 // Build X.Y.Z-R image
                 sh """
-                docker build --no-cache --tag "${params.IMAGE_NAME}:${params.FULL_VERSION}-${BUILD_NUMBER}" .
+                docker build --sbom=true --provenance mode=max,builder-id="${BUILD_URL}" --annotation "org.opencontainers.image.url=${params.REPOSITORY}" --annotation "org.opencontainers.image.source=${params.REPOSITORY}" --annotation "org.opencontainers.image.version=${params.FULL_VERSION}" --annotation "org.opencontainers.image.ref.name=${params.BRANCH}" --annotation "org.opencontainers.image.created=$(date -Iseconds)" --no-cache --tag "${params.IMAGE_NAME}:${params.FULL_VERSION}-${BUILD_NUMBER}" .
```

Comment:
I'm not sure this change is correct.

Currently, `--tag "${params.IMAGE_NAME}:${params.FULL_VERSION}-${BUILD_NUMBER}"` is present (with double quotes) and working: https://github.com/Cisco-Talos/clamav-docker/blob/fdb02aa2c20f26faeec6a6c093a881cdfce93359/clamav-bytecode-compiler/1.4/ubuntu/Jenkinsfile#L74

Yet you're saying that that line needs the double quotes changed to singles. If doubles are working now, why change to singles?

Reply:
Hi @candrews 

Thanks for asking the question. 

"org.opencontainers.image.created=$(date -Iseconds)" 
This is failing to be interpolated by Jenkins Groovy. This should be in single quotes so that only shell can interpolate it and not the Jenkins. 

Ideally, there should be a clear demarcation of what is being interpolated by groovy and shell. I will make the changes to it once your changes are merged. 

Also,
--annotation org.opencontainers.image.created="\$(date -Iseconds)" - We can do this as well to escape the "$" too to be interpolated by Groovy. 

To be consistent, I prefer to use single quotes. 

Let me know if it makes sense. 

Thanks